src/host/ft_host/InitTests.cpp

```diff
@@ -269,15 +270,18 @@ MODULE_SETUP(ModuleSetup)
         {
             auto gle = GetLastError();
             VERIFY_ARE_EQUAL(6u, gle, L"If we fail to set up the console, GetLastError should return 6 here.");
-            Sleep(1000);
+
+            // Sleep with a backoff, to give us longer to try next time.
+            Sleep(1000 * (1 + tries));

```

Comment:
Using exponential backoff with 5 retries would probably be a good idea anyways though. A good initial delay would probably be something like 10ms?, so that it doesn't throttle the tests too much on our fast developer machines. (Could you test on your own PC how many ms are enough so that it never hits retry when running locally?) Something like this:

```cpp
DWORD retries = 0;
for (DWORD delay = 10;; delay *= 2, retries++)
{
    // ...
    if (succeeded)
    {
        break;
    }

    WaitForSingleObject(GetCurrentThread(), delay);
}
```

BTW should we really reopen the CRT handles inside the loop? Seems like we should only do that once before the loop...